x/pdv/types/msg.go

```diff
@@ -7,15 +7,17 @@ import (
 
 // MsgCreatePDV defines a CreatePDV message
 type MsgCreatePDV struct {
-	Owner sdk.AccAddress `json:"owner"`
-	ID    uint64         `json:"id"`
+	Owner    sdk.AccAddress `json:"owner"`
+	Receiver sdk.AccAddress `json:"receiver"`
+	Reward   uint64         `json:"reward"`
```

Comment:
Looks like we shouldn't remove ID field

This is a ledger and I think it's fair to keep a link to pdv there

Reply:
Indeed, probably we will add some cli/rest to query them